Python/bytecodes.c

```diff
@@ -2642,14 +2631,15 @@ dummy_func(
                     PyObject *next = PyList_GET_ITEM(seq, it->it_index++);
                     PUSH(Py_NewRef(next));
                     JUMPBY(INLINE_CACHE_ENTRIES_FOR_ITER);
-                    DISPATCH();
+                    goto end_for_iter_list;  // End of this instruction
```

Comment:
:(

Reply:
Yeah, I'm really counting on the compiler to optimize this for us.